lib/server/continuation_point_manager.js

```diff
@@ -2,10 +2,9 @@
  * @module opcua.server
  */
 require("requirish")._(module);
```

Comment:
Also I think I should re check all my super calls.  I've done super(...arguments) but not sure that is right